main.cpp

```diff
@@ -47,58 +52,53 @@ int main(int argc, char *argv[]) {
 
     int mode = atoi(argv[2]);
     std::string JASMINEGRAPH_HOME = Utils::getJasmineGraphHome();
-    std::string profile = argv[1];  // This can be either "docker" or "native"
+    jasminegraph_profile = strcmp(argv[1], "docker") == 0 ? PROFILE_DOCKER : PROFILE_K8S;
     std::string enableNmon = "false";
+    main_logger.info("Using JASMINE_GRAPH_HOME=" + JASMINEGRAPH_HOME);
```

Comment:
Can we do similar refactoring for the rest of the log lines in this file? What I mean is the change `main_logger.log("", "info");` to `main_logger.info("");`. Can we do similar refactoring for other log lines as well?

Reply:
There are no occurrences of `main_logger.log()` remaining.